alpine/5/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG INVOICENINJA_VERSION
 ARG REPOSITORY=invoiceninja/invoiceninja
 ARG FILENAME=react-invoiceninja.tar
 
-RUN set -eux; apk add curl unzip grep
+RUN set -eux; apk add curl unzip grep vim nano
```

Comment:
vim/nano not required for production, you may need these whilst debugging, but they should be removed for the production builds

Reply:
no problem to remove these unnecessary cmd.